languagetool-language-modules/ga/src/main/resources/org/languagetool/resource/ga/irish_tags.txt

```diff
@@ -0,0 +1,3717 @@
+Abr
+Abr:Title
+Adj:Base
+Adj:Base:DeNom
+Adj:Base:DeNom:Com:Sg
+Adj:Base:DeNom:DefArt
+Adj:Base:DeNom:Ecl
+Adj:Base:DeNom:Gen:Pl:Ecl
+Adj:Base:DeNom:Gen:Sg:Len
+Adj:Base:DeNom:Len
+Adj:Base:Ecl
+Adj:Base:hPref
+Adj:Base:Len
+Adj:Com:NotSlen:Pl
+Adj:Com:Pl
+Adj:Com:Slender:Pl
+Adj:Com:Slender:Pl:Len
+Adj:Comp
+Adj:Comp:Len
+Adj:Fem:Com:Sg
+Adj:Fem:Com:Sg:Len
+Adj:Fem:Gen:Sg
+Adj:Fem:Voc:Sg
+Adj:Fem:Voc:Sg:Len
+Adj:Gen:Strong:Pl
+Adj:Gen:Weak:Pl
+Adj:Its
+Adj:Masc:Com:Pl
+Adj:Masc:Com:Sg
+Adj:Masc:Com:Sg:Len
+Adj:Masc:Gen:Sg
+Adj:Masc:Gen:Sg:Len
+Adj:Masc:Voc:Sg
+Adj:Masc:Voc:Sg:Len
+Adj:Voc:Pl
+Adv:Dir
+Adv:Dir:Len
+Adv:Gn
+Adv:Its
+Adv:Loc
+Adv:Q
+Adv:Q:Past
+Adv:Temp
+Adv:Temp:Gen:Sg
+Art:Gen:Sg:Def:Fem
+Art:Pl:Def
+Art:Sg:Def
+CC:Cop:Pro:Q
+CC:Noun:Masc:Com:Pl
```

Comment:
Have you considered using single letters like:

N:M   instead of "Noun:Masc"

For the language I maintain, I try to use single letters: it makes writing regexp easier in some cases and faster.
For example, in French, I want a masculine or epicene nouns, can wrote:

N [me]

instead of something like:

Noun:(maculine|epicene)

If I want to match a noun (N) or adjective (J) without caring about gender I can write:

[NJ] . 
(the dot matches any gender m, f or e for masculine, femine or epicene)